frontend/src/pages/UserView.js

```diff
@@ -134,11 +141,12 @@ function UserView() {
 			id={id}
 			select
 			label={label}
-			variant="outlined"
+			variant="standard" //"outlined"
+			fullWidth
```

Comment:
anywhere you see a change like this from variant = outlined to standard
that is a GUI change